rfc/0053-sdk3-crud.md

```diff
@@ -1188,9 +1188,18 @@ public interface ILookupInResult : IResult {
 }
 ```
 
-- Exists(..) will return true if the item exists for a given index, otherwise false.
-
-- ContentAs will throw a NotFoundException if the item does not exist; if the index is invalid a InvalidIndexException will be thrown.
+- Exists
+  1. Throws InvalidIndexException if the index is < 0 or >= the number of lookups.
+  2. Returns true if the status code at the index is `SUCCESS`.
+  3. Returns false if the status code at the index is `PATH_NOT_FOUND`.
+  4. For all other status codes, throws an exception matching the status code. See [RFC-0058](0058-error-handling.md) for error definitions.
+
+- ContentAs
+  1. Throws InvalidIndexException if the index is < 0 or >= the number of lookups.
+  2. Gets the value at the index, in one of two ways depending on the lookup type:
+     1. If the result came from an `exists` lookup, calls Exists to get a boolean, and uses the JSON representation of that boolean as the "value" in the next step. Propagates any exceptions thrown by `Exists`.
+     2. Otherwise, uses the value returned by the server as the "value" in the next step. If the status code at the index is not `SUCCESS` or `SUCCESS_DELETED_DOCUMENT`, throws an exception matching the status code. See [RFC-0058](0058-error-handling.md) for error definitions. 
```

Comment:
I think this SUCCESS_DELETED_DOCUMENT falls under the same mechanism as above?

Reply:
Yes. Thanks. Updated.